### PR TITLE
Clean up uniswap-v2 pallet.

### PR DESCRIPTION
**Pull request by vivekvpandya**

### CODE REVIEW

**Comment by KaiserKarel** [comment]
Probably good to wait on https://github.com/ComposableFi/composable/pull/460, and rebase then, as it changes some stuff as well.

**Review by dzmitry-lahoda — DISMISSED** [review]

File: frame/uniswap-v2/src/lib.rs [hunk 1]
```diff
@@ -58,52 +58,15 @@ pub mod pallet {
...
-				<Sr25519Signature as Verify>::Signer,
-				Sr25519Signature,
-			> for TestAuthId
-		{
-			type RuntimeAppPublic = Public;
-			type GenericSignature = sp_core::sr25519::Signature;
-			type GenericPublic = sp_core::sr25519::Public;
-		}
-	}
-
```
Comment (cor) [inline frame/uniswap-v2/src/lib.rs:106]: Is this module intentionally removed?

Comment (KaiserKarel) [reply frame/uniswap-v2/src/lib.rs:106]: yeah, it was there in case we needed it later, so better to reduce the attack surface and remove it

**Review by cor — APPROVED** [review]
lgtm

**Review by benluelo — APPROVED** [review]

**Review by dzmitry-lahoda — APPROVED** [review]

**Pull request merged**